test/balt.mk

```diff
@@ -6,6 +6,6 @@ $(TARGETS): $(TDIRS)
 
 $(TDIRS):
 	mkdir -p out/$@
-	$(MAKE) -C out/$@ -f ../../balt-sub.mk tdir=$@ $(MAKECMDGOALS)
+	$(MAKE) -C out/$@ -f ../../balt-sub.mk balt_files="$(addprefix "../../" $(wildcard ../compiler/testSuite/*.balt))" tdir=$@ $(MAKECMDGOALS)
```

Comment:
Can't we just do pass along TDIRS e.g. `balt_dirs=$(TDIRS)`

Reply:
fixed. `TDIRS` is only the name of directories with balt files.  We need a path to those directories since balt-sub.mk don't know whether we called it from test directory (in this case balt files are in testSuite directory) or conformance directory (in this case tests are in conformance/tests directory).